src/overview/overview.html

```diff
@@ -4,5 +4,8 @@
 <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
 
 <div id="app" />
+<div id="dab" />
```

Comment:
Should be no need to change the base HTML file; React will handle rendering out the JSX markup "shadow DOM" into the real DOM. 
The `/overview/overview.js` script tag will take care of bringing in `shareButtons.js` as it's all compiled down in the end to the single `/overview/overview.js` script

Reply:
yeah,i forgot to remove it sorry for that,i'llremove that extra div file...